test/unit/helpers/dom/PrimaryAuthForm.js

```diff
@@ -66,6 +66,11 @@ define(['jquery', 'underscore', './Form'], function ($, _, Form) {
       return this.tooltipText(SECURITY_BEACON);
     },
 
+    securityImageTooltipDestroyed: function () {
```

Comment:
nit can we change this to isSecurityImageTooltipDestroyed() and make sure it returns a boolean. Would that be better than expecting an undefined for success?